common/config.py

```diff
@@ -192,6 +192,9 @@ class Config(configfile.ConfigFileWithProfiles):
     PLUGIN_MANAGER = pluginmanager.PluginManager()
 
     def __init__(self, config_path=None, data_path=None):
+        # Note: The main profiles name here is translated using the systems
+        # current locale because the language code in the config file wasn't
+        # read yet.
         configfile.ConfigFileWithProfiles.__init__(self, _('Main profile'))
```

Comment:
In the future we should remove the ability to translate the first profiles name. A name as an identifier shouldn't be translated at all. Related to #1371